lifecycle-operator/main.go

```diff
@@ -273,16 +274,16 @@ func main() {
 
 	workloadVersionLogger := ctrl.Log.WithName("KeptnWorkloadVersion Controller").V(determineWorkloadVersionControllerLogLevel(env.KeptnWorkloadVersionControllerLogLevel, env.KeptnWorkloadInstanceControllerLogLevel))
 	workloadVersionRecorder := mgr.GetEventRecorderFor("keptnworkloadversion-controller")
-	workloadVersionReconciler := &keptnworkloadversion.KeptnWorkloadVersionReconciler{
-		SchedulingGatesHandler: controllercommon.NewSchedulingGatesHandler(mgr.GetClient(), workloadVersionLogger, env.SchedulingGatesEnabled),
-		Client:                 mgr.GetClient(),
-		Scheme:                 mgr.GetScheme(),
-		Log:                    workloadVersionLogger,
-		EventSender:            controllercommon.NewEventMultiplexer(workloadVersionLogger, workloadVersionRecorder, ceClient),
-		Meters:                 keptnMeters,
-		TracerFactory:          telemetry.GetOtelInstance(),
-		SpanHandler:            spanHandler,
-	}
+	workloadVersionReconciler := keptnworkloadversion.NewReconciler(
+		mgr.GetClient(),
+		mgr.GetScheme(),
+		controllercommon.NewEventMultiplexer(workloadVersionLogger, workloadVersionRecorder, ceClient),
+		workloadVersionLogger,
+		keptnMeters,
+		spanHandler,
+		telemetry.GetOtelInstance(),
+		controllercommon.NewSchedulingGatesHandler(mgr.GetClient(), workloadVersionLogger, env.SchedulingGatesEnabled),
+		otel.Tracer("keptn/workloadversion-controller"))
```

Comment:
why do we need an additional otel tracer here ?

Reply:
It part of the parameter required by the function `NewReconciler`

Reply:
let's just keep it simple and use `keptnworkloadversion.KeptnWorkloadVersionReconciler{}` as before. There is no need to introduce a new `NewReconciler` function here

Reply:
Alright